src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml

```diff
@@ -144,6 +157,10 @@ mb:
         description: Element über das eine Treffpunkt generiert und verschickt werden kann.
       label:
         text: Bearbeiten Sie den Text
+      admin:
+        body: Körper
```

Comment:
"Körper" sagt man in dem Kontext nicht auf deutsch. "Inhalt" oder einfach "Text" würde es besser treffen

Reply:
Text passt doch gut

Reply:
nein. Hier geht es um den HTML-Inhalt, der beim Senden der Nachricht eines POI gesendet wird, das heißt im Deutschen nicht Körper.